service/history/handler.go

```diff
@@ -1136,6 +1136,11 @@ func (h *Handler) DeleteWorkflowExecution(ctx context.Context, request *historys
 
 	workflowExecution := request.WorkflowExecution
 	workflowID := workflowExecution.GetWorkflowId()
+	h.logger.Debug("DeleteWorkflowExecution requested",
```

Comment:
Use Info level instead of debug level. It is unusual to call API to delete a workflow.

Reply:
And I would move to after the 2 error condition check before the engine.DeleteWorkflowExecution() call.

Reply:
thanks, updated!